### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by shehab6280** [johngodley/redirection#680]
```
Plugin: 3.0.1
WordPress: 4.9.2
PHP: 5.6.33
Browser: Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36 OPR/50.0.2762.67

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/setting GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /wp-json/redirection/v1/setting was not found on this server.</p>
<p>Additionally, a 404 Not Found
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

```

**Comment by johngodley** [comment]
Can you try using the 3.1 beta?

https://github.com/johngodley/redirection/releases/tag/3.1

I will be releasing this version tomorrow if you want to wait until then.

**Comment by johngodley** [comment]
I've now released 3.1. I will close this ticket for now, but please reopen if the problem persists
